.chloggen/mdatagen-logs-flag.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: cmd/mdatagen
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Introduce `logs.enabled: <bool>` flag to enable basic logs' builder."
```

Comment:
Why do we need `logs.enabled`?

Reply:
I think it's better to have a flag in metadata.yaml to control the generation of log builder-related files, rather than relying on the supported signal. I've removed it from this PR, but I can add it in a follow-up PR if needed.

Reply:
I think as long as logs are supported the basic logs builder can be generated